doc/rust.txt

```diff
@@ -157,6 +157,14 @@ COMMANDS                                                       *rust-commands*
 		If |g:rustc_path| is defined, it is used as the path to rustc.
 		Otherwise it is assumed rustc can be found in $PATH.
 
+:Play                                                                  *:Play*
+        This command is only available if you have web-api.vim[0]
+        installed.  It sends the current selection, or if nothing is
+        selected, the entirety of the current buffer to the Rust playpen,
+        and emits a message with the is.gd shortened URL to the playpen.
+
+        [0]: https://github.com/mattn/webapi-vim
+
```

Comment:
This file is (as is standard for Vim help) formatted with eight-character-wide tabs; your four spaces will need to be replaced and the paragraph reflowed to the textwidth of 78. The Markdown-like formatting of the link doesn’t really feel quite right either; how about something like this for the whole thing (looks like GitHub replaces my literal tabs with four spaces; not good; ah well, that part will need fixing):

```
:Play                                                                  *:Play*
		This command is only available if you have web-api.vim
		installed (available at https://github.com/mattn/webapi-vim).
		It sends the current selection, or if nothing is selected, the
		entirety of the current buffer to the Rust playpen, and emits
		a message with the is.gd shortened URL to the playpen.

		|g:rust_playpen_url| is the base URL to the playpen, by
		default "https://play.rust-lang.org/".

```

I also added mention of `g:rust_playpen_url`; it feels like it should be mentioned.